src/crimson/osd/pg.cc

```diff
@@ -931,12 +935,18 @@ seastar::future<> PG::handle_rep_op(Ref<MOSDRepOp> req)
 void PG::handle_rep_op_reply(crimson::net::Connection* conn,
 			     const MOSDRepOpReply& m)
 {
-  if (is_valid_rep_op_reply(m)) {
+  if (!can_discard_replica_op(m)) {
```

Comment:
could you please extract the rename change into a separate commit ?

Reply:
ok, that shall be done:-)